.github/workflows/build.yml

```diff
@@ -313,14 +318,13 @@ jobs:
           echo "Running integration tests with tags: $TEST_SUITE_TAGS (empty means every test not specifically tagged)"
 
           export TEST_SUITE_TAGS="$TEST_SUITE_TAGS"
-          TIMEOUT=15m
+          TIMEOUT=30m
```

Comment:
I was running into this limit because I was tagging multiple tests. So the 15m timeout is fine if you're just running a test label or two, but if you're running more than that you'll likely run into a timeout.